### DISCUSSION TITLE
Light Card

### DISCUSSION THREAD

**Discussion by hyperfive** [piitaya/lovelace-mushroom#437] [Ideas]
It would be awesome feature if when a user enables color control, temperature control, and brightness control they had the option of displaying individual sliders for each of them vs hiding color & temperature behind icons.  Additionally if the user decides to combine multiple settings into the same slider like the current default, the user should be able to select the slider that is the main control surface vs only brightness being the option.

Here is the current mode of when all three are enabled 

<img width="449" alt="Screen Shot 2022-05-02 at 7 32 20 PM" src="https://user-images.githubusercontent.com/46284239/166342645-50043dd7-8903-4161-97ef-24331e1c0d12.png">

Here is the idea of the desired state (please excuse low res mockup)

![Mushroom Light Card Idea](https://user-images.githubusercontent.com/46284239/166343169-c91c31e6-57d8-45fb-ae46-4bac0ea6e14f.png)

**Comment by johannes-z** [comment]
Agreed, or maybe even move the buttons to swap the controls to the top right of the card, so the sliders are always at max width. If you use these controls with a horizontal stack, the sliders become so tiny that they are practically unusable because they lack fine control.

**Comment by piitaya** [comment]
I think we can combine the 2 buttons in 1. But the controls will stay at the bottom of the card and I prefer to not allow multiple line of controls de keep it simple and minimalist. If you need more control, you can use the more-info popup.

**Comment by zgadson** [comment]
First off, your Mushroom Cards are awesome. Thank you for making them available to everyone!

Came here to ask this question - not to move anything in the card layout - but to ask if a marker can be added to light sliders like in the color sliders. Would be awesome if they matched!

**Reply by piitaya** [reply]
Like this ?
![CleanShot 2023-02-03 at 18 28 26](https://user-images.githubusercontent.com/5878303/216668347-ea8a6671-2b98-4ed9-9e22-565baf7e30cb.png)

**Reply by zgadson** [reply]
Exactly! How can I make mine look like this, too?

**Reply by piitaya** [reply]
It's the tile card: https://www.home-assistant.io/dashboards/tile/
But I plan to add the same design in Mushroom.

**Reply by zgadson** [reply]
Thank you! That does work to get the marker to appear, but it seems the Tile card lacks some functionality I need; like collapsing the card when not in use, and hiding everything but the slider itself.

![unnamed](https://user-images.githubusercontent.com/68963862/216676498-5816eea7-2106-49b0-b7ab-f4df81da3391.jpg)

**Comment by zgadson** [comment]
Hi Piitaya! Is it possible to hide elements of your Tile Card? In particular, the icon, name, and label, so it only shows the slider?

**Reply by piitaya** [reply]
No that's not possible 😕
